### DISCUSSION TITLE
My Webpage has custom fonts but pages is not displaying them.

### DISCUSSION THREAD

**Discussion by hethwiQ** [community/community#82387] [Other Feature Feedback, Questions, & Ideas]
Question

### Body

Greetings one and all,
My webpage [https://hethwiq.github.io/Port/](url) has a custom font. but for some reason it won't display.
Here's my repo [https://github.com/hethwiQ/Port/](url). Some of them are displaying while some aren't. It'd be helpful for finding the solution.
![image](https://github.com/community/community/assets/80631564/8a64c93b-8d62-464a-b2e9-dfd853db50fa)

many thanks.

**Comment by yoannchaudet** [comment]
I think you figured it out because it now looks great and I don't see 404 errors when loading your fonts :)

<img width="1747" alt="image" src="https://github.com/community/community/assets/14911070/6c6afcbc-7a1b-4c7c-80d6-0d49d466a6f1">

**Reply by hethwiQ** [reply]
Thankyou for your reply, I didn't yet figure it out, i just made some random changes for the path to the font files, i didn't keep the track of what changes worked. Explaining would be helpful.
[https://hethwiq.github.io/Port/](url)
Thanks for your compliment "now it looks great" :)

**Reply by yoannchaudet** [reply]
A common source of errors with GitHub Pages when developing locally vs a deployed website is about "path based routing".

If you name your repository `<your username>.github.io`, then your Pages site will automatically be hosted at `https://<your username>.github.io`. In this configuration both local and deployed site will behave the same. A picture for instance `logo.jpg` will be served and available at `/logo.jpg`.

If you name your repository any other way (like in your example `Port`). Then your site will deployed under `https://<your username>.github.io/Port`. That means you need to prefix with `/Port` all the files you want to reference in your HTML files. The logo from previously will only be available under `/Port/logo.jpg`. If you craft a website with HTML directly, you need to be aware of this path. If you use a static site generator, usually they have an option somewhere to configure a prefix path to use.

Hope that helps!

**Reply by yoannchaudet** [reply]
And it was probably much better explained right here: https://github.com/orgs/community/discussions/61073 ❤️

**Reply by hethwiQ** [reply]
Thankyou for the clarification, this is very helpful.
